templates/certificates/_accomplishment-banner.html

```diff
@@ -5,35 +5,20 @@
 <%namespace name='static' file='../static_content.html'/>
 <%block name="js_extra">
   <%static:js group='certificates_wv'/>
+  <script type="text/javascript" src="https://mandrillapp.com/api/docs/js/mandrill.js"></script>
+  <script type="text/javascript" src="${static.url('themes/extracare-dogwood/js/email_cert.js')}"></script>
   <script type="text/javascript">
-      $(document).ready(function() {
-          FaceBook.init({"facebook_app_id": '${facebook_app_id}'});
-          $.ajaxSetup({
-              headers: {
-                  'X-CSRFToken': $.cookie('csrftoken')
-              },
-              dataType: 'json'
-          });
-          $(".action-linkedin-profile").click(function() {
-              var data = {
-                  user_id: '${accomplishment_user_id}',
-                  course_id: $(this).data('course-id'),
-                  enrollment_mode: $(this).data('certificate-mode'),
-                  certificate_id: '${certificate_id_number}',
-                  certificate_url: window.location.href,
-                  social_network: 'LinkedIn'
-              };
-              Logger.log('edx.certificate.shared', data);
-              window.open('${linked_in_url}');
-          });
-      });
-
-      function popupWindow(url, title, width, height) {
-          // popup a window at center of the screen.
-          var left = (screen.width/2)-(width/2);
-          var top = (screen.height/2)-(height/2);
-          return window.open(url, title, 'toolbar=no, location=no, directories=no, status=no, menubar=no, scrollbars=yes, resizable=yes, width='+width+', height='+height+', top='+top+', left='+left);
-      }
+      var subject = decodeURIComponent("${email_share_subj}");
+      var body = decodeURIComponent("${email_share_body}");
+      var to = decodeURIComponent("${email_share_to}");
+      var params = {
+          "message": {
+              "from_email":"technical+extracare@appsembler.com",
+              "to":[{"email":"amir@appsembler.com"}],
```

Comment:
test data still in here  `amir@appsembler.com`

Reply:
`from_email` looks good.  Still needs to get the proper `StudyLocation` email address instead of `amir@appsembler.com`